fitnessllm_dataplatform/http_handler.py

```
@@ -0,0 +1,100 @@
+            self.wfile.write(json.dumps({"error": str(e)}).encode())
+        except Exception as e:
+            self.send_response(500)
+            self.send_header("Content-type", "application/json")
+            self.end_headers()
+            self.wfile.write(json.dumps({"error": str(e)}).encode())
+
+
+def run_server(port: int = 8080) -> None:
+    """Run the HTTP server."""
+    port = int(os.environ.get("PORT", 8080))
+    server = HTTPServer(("0.0.0.0", port), Handler)
```

Ruff (0.8.2)

Possible binding to all interfaces

(S104)